client/src/components/NavBar.js

```diff
@@ -38,17 +39,34 @@ class NavBar extends Component {
       logout,
       handleSidebarHide
     } = this;
+    const landingPage = window.location.pathname === "/";
```

Comment:
You do not need this after the update I suggested it.

Reply:
ok I have done